### DISCUSSION TITLE
Clarity on how to get a ref to the editor

### DISCUSSION THREAD

**Discussion by scrubmunch** [steven-tey/novel#244] [General]
Is it possible to get a ref to the editor when using the npm package? I've seen people using editor.commands. etc but as far as I can tell there's no clear way of exposing the tiptop editor. I'm a beginner so any help is appreciated!

**Comment by FranciscoMoretti** [comment]
Not that I know of. The `onCreate` method from tiptap editor could be exposed by `NovelEditor` so that you can save the editor with it. I can do a PR if this is something desired by the project.

**Reply by scrubmunch** [reply]
It's certainly something I would find useful! I'm unsure if many others would, but I assume it would be nice to have easy access to the tiptap editor for customizing it and expanding the use cases.
